api/src/main/java/org/apache/iceberg/actions/RewriteDataFiles.java

```diff
@@ -171,6 +171,16 @@ default RewriteDataFiles zOrder(String... columns) {
    */
   RewriteDataFiles filter(Expression expression);
 
+  /**
+   * Specify the branch on which the rewrite will be performed. via WAP(write-audit-publish)
```

Comment:
> via WAP(write-audit-publish)

I don't think this sentence is needed? And it' also incomplete sentence

Reply:
I see, looks like WAP is supported through the procedure, but I don't think we need to mention it here in the RewriteDataFiles class, since that is a caller's logic to map WAP branch to this